landscape/lib/apt/package/store.py

```diff
@@ -450,23 +452,12 @@ def remove(self, cursor):
 
 
 class PackageTask:
-    def __init__(self, db, id):
+    def __init__(self, db, cid, queue, timestamp, data):
         self._db = db
-        self.id = id
-
-        cursor = db.cursor()
-        try:
-            cursor.execute(
-                "SELECT queue, timestamp, data FROM task " "WHERE id=?",
```

Comment:
Ha looks like I missed a "could be single string" previously. Oh well, doesn't matter now.